sw/device/lib/dif/dif_i2c.c

```diff
@@ -19,7 +19,7 @@
  */
 static uint16_t round_up_divide(uint32_t a, uint32_t b) {
   if (a == 0) {
-    return 0;
+    return 1;
```

Comment:
a value of 1 hold cycles would actually cause the controller to get stuck forever.
Basically internally a -1 value is loaded and the controller just waits forever.

Reply:
Hm... instead of changing this function to compute the arithmetic differently from its name, should we just use the value `1` for `data_signal_hold_cycles` instead of 0?

I can't decide if this is an IP bug, but if it isn't, we should probably inform the user in the documentation.

Reply:
yeah i thought about that... but then i thought..if someone wanted to ever change it to non-zero or one, then they could just change that one value.

Maybe if i want to stick with this, i need to give this a better name?

I also did file #14667, because i thought this was very odd.

Reply:
you know what? screw it, let's go with 1